warwomb/utilities/seed.py

```diff
@@ -0,0 +1,120 @@
+import os
+import json
+import sqlalchemy as db
+from dotenv import main
+
+main.load_dotenv()
+
+db_engine = None
+db_metadata = None
+unit_factions_table = None
+unit_faction_links_table = None
+units_table = None
+
+faction_names_map = {
+    "AC": { "dbid": None, "name": "Aeternus Continuum" },
+    "EMP": { "dbid": None, "name": "Empyrean" },
+    "ISA": { "dbid": None, "name": "Iron Star Alliance" },
+    "MW": { "dbid": None, "name": "Marcher Worlds" }
+    }
+
+# TODO: add 'tbd' columns to unit data file
+unit_column_map = {
+    "name": "name",
+    "tbd": "speed",
+    "tbd": "strength",
+    "tbd": "melee_attack",
+    "tbd": "ranged_attack",
+    "tbd": "defense",
+    "tbd": "armor",
+    "tbd": "focus",
+    "tbd": "health",
+    "dc": "deployment_cost"
+}
+
+data_file = open('../data_files/warcaster_units.txt', 'r')
+unit_dictionary = json.loads(data_file.read())
+data_file.close()
+
+def create_db_engine():
+    global db_engine, db_metadata
+    db_engine = db.create_engine(os.getenv("DATABASE_URL"))
+    connection = db_engine.connect()
+    db_metadata = db.MetaData()
+    connection.close()
+
+def setup_tables():
+    global unit_faction_links_table,  unit_factions_table, units_table
+    unit_faction_links_table= db.Table('unit_faction_links', db_metadata, db.Column("unit_id", db.Integer), db.Column("unit_faction_id", db.Integer))
+    unit_factions_table = db.Table('unit_factions', db_metadata, db.Column("id", db.Integer, primary_key=True, autoincrement=True, nullable=False), db.Column("name", db.String))
+    units_table = db.Table('units', db_metadata, 
+        db.Column("id", db.Integer,  primary_key=True, autoincrement=True, nullable=False),
+        db.Column("name", db.String),
+        db.Column("speed", db.Integer),
+        db.Column("strength", db.Integer),
+        db.Column("melee_attack", db.Integer),
+        db.Column("ranged_attack", db.Integer),
+        db.Column("defense", db.Integer),
+        db.Column("armor", db.Integer),
+        db.Column("focus", db.Integer),
+        db.Column("health", db.Integer),
+        db.Column("deployment_cost", db.Integer))
+
+def reset_tables():
```

Comment:
This can be accomplished with an [alembic downgrade](https://alembic.sqlalchemy.org/en/latest/tutorial.html#downgrading). Specifically, `python -m alembic downgrade base` ought to do what you're doing here.